tests/test_0001_html.py

```diff
@@ -336,3 +337,46 @@ def test_html_strings():
     node = HTML('<button disabled="true">Click me</button>')[0]
 
     assert node.disabled
+
+    # data binding nodes ######################################################
+    node = HTML('<input value="abc"/>')[0]
+
+    assert type(node) is TextInput
+    assert node.value == 'abc'
+
+    node = HTML('<input type="text" value="xyz" disabled/>')[0]
+
+    assert type(node) is TextInput
+    assert node.value == 'xyz'
+    assert node.disabled == True  # NOQA: E712
```

Comment:
I assume you want to check if the type of ``node.disable`` is a boolean. If you use ``is`` instead of ``==`` you don't have to add the ``NOQA``

Reply:
Good catch, thanks!